core/src/main/resources/application.conf

```diff
@@ -118,6 +120,11 @@ call-caching {
   // request to an external service (DockerHub, GCR).  If a call fails to lookup a
   // Docker hash, it will fail.
   lookup-docker-hash = false
+
+  // This option determines whether file contents are hashed to determine cache hit or miss.
+  hash-file-contents = true
+  // This option determines whether file paths are hashed to determine cache hit or miss.
+  hash-file-paths = false
```

Comment:
Ok, I can buy that.  So for the Docker case, if we have:

```
call-caching {
  docker-hashing: "lookup"   # Could be "name"
}
```

In Docker "lookup" mode, Cromwell could write one aggregate hash using the Docker image name but *not* the Docker image hash, and a second aggregate hash using the Docker image hash but *not* the Docker image name.  If a subsequent call reads the cache in Docker "lookup" mode, its Docker image hash would be included in its aggregate hash for cache hit determination, but not its name.  In "name" mode, the opposite would be true.

Or do we really want to support both Docker name and image hashes as part of one aggregate hash?